doc/source/conf.py

```diff
@@ -129,13 +129,13 @@
 # html_theme_options = {}
 html_theme_options = {
     'logo': 'logo.png',
-    'github_user': 'metacloud',
+    'github_user': 'ansible',
     'github_repo': 'molecule',
     'github_button': True,
     'travis_button': False,
```

Comment:
```suggestion
    'travis_button': True,
```

Reply:
Nice, also added `'badge_branch': 'master',` as it wasn't clear on https://alabaster.readthedocs.io/en/latest/customization.html what default is used, and feels better to be explicit

Reply:
They linked this file with defaults from docs: https://github.com/bitprophet/alabaster/blob/master/alabaster/theme.conf

But yes, explicit is better than implicit :)